packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts

```diff
@@ -9,6 +10,10 @@ export const isRangeFacetValueSelected = (selection: RangeFacetValue) => {
   return selection.state === 'selected';
 };
 
+export const isRangeFacetValueExcluded = (selection: RangeFacetValue) => {
```

Comment:
I think you forgot to add this method to the `headless-core-range-facet`

Reply:
Whoops,  good catch! I'll add that now ✅